code/game/machinery/vendor/weapon_kit_vendors.dm

```diff
@@ -134,13 +129,12 @@
 
 /obj/machinery/vending/blackshield_kit/proc/RedeemSecondary(obj/item/voucher/voucher, mob/redeemer)
 	var/items = list(
-					"Stallion Kit" = /obj/item/storage/box/bs_kit/stallion,
 					"Colt Kit" = /obj/item/storage/box/bs_kit/colt,
-					"Makarov Kit" = /obj/item/storage/box/bs_kit/makarov,
+					"Makarov Kit" = /obj/item/storage/box/bs_kit/makarov,		
+					"Stallion Kit" = /obj/item/storage/box/bs_kit/stallion,
 					"Cowboy Kit" = /obj/item/storage/box/bs_kit/rex10,
 					"Pilgrim Kit" = /obj/item/storage/box/bs_kit/pilgrim,
-					"Sawn-Off Shotgun Kit" = /obj/item/storage/box/bs_kit/sawn_shotgun,
-					"Martin Kit" = /obj/item/storage/box/bs_kit/martin)
```

Comment:
Martin should stay

Reply:
I guess if you want it to stay just feels weird to have 2 of them